iowrite.cpp

```diff
@@ -173,20 +174,21 @@ bool writeVelocityDistributionData(const uint popID,Writer& vlsvWriter,
    // space, but a purely numerical bounding box.
    uint64_t bbox[6];
    const size_t meshID = getObjectWrapper().particleSpecies[popID].velocityMesh;
-   bbox[0] = getObjectWrapper().velocityMeshes[meshID].gridLength[0];
-   bbox[1] = getObjectWrapper().velocityMeshes[meshID].gridLength[1];
-   bbox[2] = getObjectWrapper().velocityMeshes[meshID].gridLength[2];
-   bbox[3] = getObjectWrapper().velocityMeshes[meshID].blockLength[0];
-   bbox[4] = getObjectWrapper().velocityMeshes[meshID].blockLength[1];
-   bbox[5] = getObjectWrapper().velocityMeshes[meshID].blockLength[2];
+   bbox[0] = vmesh::getMeshWrapper()->velocityMeshes->at(meshID).gridLength[0];
+   bbox[1] = vmesh::getMeshWrapper()->velocityMeshes->at(meshID).gridLength[1];
+   bbox[2] = vmesh::getMeshWrapper()->velocityMeshes->at(meshID).gridLength[2];
+   bbox[3] = vmesh::getMeshWrapper()->velocityMeshes->at(meshID).blockLength[0];
+   bbox[4] = vmesh::getMeshWrapper()->velocityMeshes->at(meshID).blockLength[1];
+   bbox[5] = vmesh::getMeshWrapper()->velocityMeshes->at(meshID).blockLength[2];
 
    attribs.clear();
    attribs["mesh"] = getObjectWrapper().particleSpecies[popID].name;
    attribs["type"] = vlsv::mesh::STRING_UCD_AMR;
 
-   // stringstream is necessary here to correctly convert refLevelMaxAllowed into a string 
+   // stringstream is necessary here to correctly convert refLevelMaxAllowed (hardcoded to zero now) into a string 
    stringstream ss;
-   ss << static_cast<unsigned int>(getObjectWrapper().velocityMeshes[meshID].refLevelMaxAllowed);
+   //ss << static_cast<unsigned int>(vmesh::getMeshWrapper()->velocityMeshes->at(meshID).refLevelMaxAllowed);
+   ss << static_cast<unsigned int>(0);
```

Comment:
That zero will be removed in due time, correct?

Reply:
This zero is what is used to store in the VLSV file that this file does not contain velocity mesh refinement. I think it's probably safer to keep storing the value in the file, like before. I can clean up the comment lines, though.